kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/SessionsResource.scala

```diff
@@ -61,6 +57,16 @@ private[v1] class SessionsResource extends ApiRequestContext {
     }
   }
 
+  @GET
+  @Path("{sessionHandle}/info/{infotype}")
```

Comment:
Keep the same code style about `sessionHandle ` and `infotype `? camelCase or not?